packages/protocol/contracts/L1/verifiers/SgxVerifier.sol

```diff
@@ -67,11 +67,17 @@ contract SgxVerifier is EssentialContract, IVerifier {
 
     /// @notice Adds trusted SGX instances to the registry.
     /// @param _instances The address array of trusted SGX instances.
-    function addInstances(address[] calldata _instances) external onlyOwner {
+    function addInstances(address[] calldata _instances)
```

Comment:
I think the code in this function can be extracted to a `_addInstances` internal method, and then we have the existing two `addInstances` reuse this internal function, but doing their specific checks first (`onlyOwner` check and the valid SGX signature check/update respectively). Just to remove some otherwise duplicated code.

Reply:
Done. Thanks ! 🖖